packages/terra-detail-view/docs/themeable-variables.md

```diff
@@ -0,0 +1,14 @@
+# Themeable Variables
+
+Variables to `theme` the terra-detail-view component.
```

Comment:
My thought is the styles for detail view don't need to be themeable.

Thoughts? @mjhenkes @jmsv6d @mndrsn